src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/NoProvidersAlert.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export default function NoProvidersAlert({ currentLocation, typeOfCareName }) {
+  const headline = `We can't find any ${typeOfCareName} providers close to you`;
```

Comment:
Generally instead of quote (`'`) marks we want to use smart quotes like `’`. There are a few quote marks in the copy in here.

Example from elsewhere in the vaos code: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/appointment-list/components/AppointmentsPage/RequestExpressCare.jsx#L34